src/monaca.js

```diff
@@ -2280,24 +2280,36 @@
     return deferred.promise;
   };
 
-  Monaca.prototype.isTranspilable = function(projectDir) {
-    var projectInfoFile = path.resolve(path.join(projectDir, '.monaca', 'project_info.json'));
-    var config = require(projectInfoFile);
-    var type = config['template-type'];
+  var project_json_data = null;
 
-    return type && ( type === 'react' || type === 'angular2' );
+  Monaca.prototype.fetchProjectData = function(projectDir) {
```

Comment:
Please do not cache this data since the file will be changed during the lifecycle of Localkit (not CLI).